### PR TITLE
Add related software to downloads page and add imaging to it

### PR DESCRIPTION
**Pull request by Pasarus**
To Test:

Build the docs and check that it looks alright locally
Ensure the Mantid imaging link takes you to the installation page for imaging

### CODE REVIEW

**Comment by Pasarus** [comment]
https://github.com/mantidproject/mantidimaging/issues/710

File: templates/downloads.html [hunk 1]
```diff
@@ -62,4 +62,13 @@ <h2>Nightly Development Builds</h2>
         <p>Previous nightly builds can be found <a href="https://sourceforge.net/projects/mantid/files/Nightly/">here</a>.</p>
       </article>
     </section>
+        <section id="gridcontainer">
+      <article id="other-software" class="column">
+        <h2>Related Software</h2>
```
Comment (martyngigg) [inline templates/downloads.html:67]: I know I suggested Related but could we call this "Other Software" and have the description read "Other software relating to neutron data reduction/analysis"?

Comment (martyngigg) [reply templates/downloads.html:67]: Would it also be difficult to move this to the right-hand side under the nightly downloads?

Comment (Pasarus) [reply templates/downloads.html]: Both are done :+1:

**Review by martyngigg — APPROVED** [review]
Thanks. 👍

**Pull request merged**